index.html

```diff
@@ -117,6 +117,227 @@ <h3 class="hero-section__subtitle">I'm a software developer</h3>
         <i class="fa-solid fa-chevron-down"></i>
       </a>
     </section>
+    <main id="recent-works" class="my-main">
+      <div class="recent-works__title--wrapper">
+        <h1 class="recent-works__title">My Recent Works</h1>
+        <hr class="recent-works__hr-line" />
+      </div>
+      <section class="recent-works projects-container container">
+        <div class="projects-card">
+          <img
+            src="src/assets/media/img-placeholder.png"
+            alt="template will be replaced by a project soon"
+            class="projects-card__img"
+          />
+          <div class="projects-card__title--wrapper">
+            <h1 class="projects-card__title">Multi-Post Stories</h1>
```

Comment:
[OPTIONAL]
- Great job on implementing the previous reviewer's suggestions :rocket: .
- [ ] I recommend using only one `<h1>` on your project, more preferably on your project's name. This will improve your project's accessibility.

Reply:
Hi @NatiG25! thank you for your review 😃!, yes I was wondering that. I had the same thought towards only one  -h1- for the whole webpage, however, there are a large number of articles that differ from each other, whether to use only one or several within different sections 😥... still, most of those articles clarify that only one is better for as you said -accessibility-. So, I will read more about it and think to implement the best on the project and also for future projects. Thank you again for your suggestion, I will take it into account 🏆.